src/Middleware/ErrorCatcher.php

```diff
@@ -120,9 +123,12 @@ public function forceContentType(string $contentType): self
 
     public function process(ServerRequestInterface $request, RequestHandlerInterface $handler): ResponseInterface
     {
+        $t = null;
         try {
             return $handler->handle($request);
         } catch (Throwable $t) {
+            $this->eventDispatcher?->dispatch(new ApplicationError($t));
```

Comment:
If here will be exception then is not catch. See https://3v4l.org/gYeWI

Reply:
https://3v4l.org/ud0Dn

Reply:
> https://3v4l.org/ud0Dn

Interesting behavior 😀

In this case suppress exception in event. It's expected behavior? We won't know about this error in any way... May be need handle exception in event?

Reply:
Good question. 
It would be good to log it at least, at most add to the response.
PHP doesn't support any compound exceptions, so it's impossible to combine the new and the old ones. As a workaround we may support several exceptions in error renders, but it's too much.

[Here](https://github.com/yiisoft/error-handler/blob/master/src/ErrorHandler.php#L201) we have a line without catching an exception, but it may be useful to render them both too.